src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_TERM = 40;
```

Comment:
Here the naming is not clear, it is better to use the following:
```suggestion
  const PRICE_PER_DAY = 40;
```